### DISCUSSION TITLE
[feature request] add generic oidc laravel plugin

### DISCUSSION THREAD

**Discussion by lknite** [linuxserver/Heimdall#987] [Ideas]
Looks like if this were installed then I'd just have to configure it:
https://github.com/furdarius/oidconnect-laravel

**Comment by lknite** [comment]
Reopening, I'm not sure if this should be a heimdall requirement or part of the docker container I usually use.  If it were here, then everyone could use it and not just the container folks.

**Comment by lknite** [comment]
I've built a pipeline and am working on this.  First working to install keycloak specific plugin, and then later a more generic plugin.  Doing this because the more generic oidc plugin which I've found requires php8 and laravel8.  Will report once I get the keycloak plugin working while waiting for php8 / laravel8.

**Comment by KodeStar** [comment]
2.5.0 will have Laravel 8
